src/main/MQ2Spawns.cpp

```diff
@@ -81,195 +72,464 @@ static bool dataNamingSpawn(const char* szIndex, MQTypeVar& Ret)
 	return false;
 }
 
-enum eCaptionColor
-{
-	CC_PC = 0,
-	CC_PCConColor = 1,
-	CC_PCPVPTeamColor = 2,
-	CC_PCRaidColor = 3,
-	CC_PCClassColor = 4,
-	CC_PCGroupColor = 5,
-	CC_PCTrader = 6,
-	CC_NPC = 7,
-	CC_NPCConColor = 8,
-	CC_NPCClassColor = 9,
-	CC_NPCMerchant = 10,
-	CC_NPCBanker = 11,
-	CC_NPCAssist = 12,
-	CC_NPCMark = 13,
-	CC_PetNPC = 14,
-	CC_PetPC = 15,
-	CC_PetConColor = 16,
-	CC_PetClassColor = 17,
-	CC_Corpse = 18,
-	CC_CorpseClassColor = 19,
-};
-
-struct CAPTIONCOLOR
+static void ConvertCR(std::string& text)
```

Comment:
yeah it started as just copying from the existing implementation and then me deciding on the spot it wasn't good enough, and that I wanted to do the replacement in place instead of copying a string.